Source/Visitable/VisitableViewController.swift

```diff
@@ -41,6 +42,14 @@ open class VisitableViewController: UIViewController, Visitable {
         visitableView.hideActivityIndicator()
     }
     
+    open func visitableDidActivateWebView(_ webView: WKWebView) {
+        // No-op
+    }
+    
+    open func visitableDidDeactivateWebView() {
+        // No-op
+    }
+    
```

Comment:
This functions are no-op by default. Subclasses of `VisitableViewController` can override them if needed.